internal/utils.go

```diff
@@ -309,6 +319,13 @@ func setTemplateOptions(tmpl map[string]interface{}, ignorePending bool, extraDe
 // - the manifest is a root policy manifest
 // - the manifest is invalid because it is missing a name
 func isPolicyTypeManifest(manifest map[string]interface{}, informGatekeeperPolicies bool) (bool, bool, error) {
+	// check for object-templates-raw separate from policies since they have separate requirements
+	_, found, _ := unstructured.NestedString(manifest, "object-templates-raw")
+	if found {
+		// return true for isPolicyType, since object-templates-raw is in a ConfigurationPolicy
+		return true, false, nil
```

Comment:
The second boolean, whether it's an OCM policy, should also be `true` so that it gets passed through the `setTemplateOptions()` function.
```suggestion
		return true, true, nil
```